chia/server/server.py

```diff
@@ -407,9 +407,6 @@ async def start_client(
                 if is_feeler:
                     asyncio.create_task(connection.close())
                 return True
-            else:
-                await session.close()
-                return False
```

Comment:
I turned this into an early exit, saving one level of indentation for a lot of lines